gulpfile.js

```diff
@@ -29,6 +38,68 @@ gulp.task('templates', function() {
     .pipe(gulp.dest(outputPath + 'js'));
 });
 
+/*********************************************************
+ * JavaScript
+ ********************************************************/
+
+gulp.task('js:build', function() {
+  return browserify(projectPath + 'app.js', {
+    debug: false
+  })
+    .transform(babelify, {
+      presets: ['es2015']
+    })
+    .bundle()
+    .pipe(source('bundle.js'))
+    .pipe(buffer())
+    .pipe(ngAnnotate())
+    .pipe(uglify())
+    .pipe(gulp.dest(outputPath + 'js'));
+});
+
+gulp.task('js:dev', function() {
+  return browserify(projectPath + 'app.js', {
+    debug: true
+  })
+    .transform(babelify, {
+      presets: ['es2015']
+    })
+    .bundle()
+    .pipe(source('bundle.js'))
+    .pipe(gulp.dest(outputPath + 'js'));
+});
+
+gulp.task('js:watch', [], function() {
+  var bundler = browserify({
+    entries: [projectPath + 'app.js'],
+    extensions: ['.js'],
+    debug: true,
+    fullPaths: true,
+    cache: {},
+    packageCache: {}
+  })
+    .transform('babelify', {
+      presets: ['es2015']
+    });
+
+  var rebundle = function() {
+    var startDate = new Date();
```

Comment:
Ok cool, can we remove it since the commit provides the information about the performance of the change and does not provide any value to the dev process.

Thanks.